tests/custom_lookups/tests.py

```diff
@@ -544,19 +544,19 @@ class CustomisedMethodsTests(SimpleTestCase):
 
     def test_overridden_get_lookup(self):
         q = CustomModel.objects.filter(field__lookupfunc_monkeys=3)
-        self.assertIn('monkeys()', str(q.query))
+        self.assertRegex(str(q.query), r"'?monkeys'?\(\)", str(q.query))
 
     def test_overridden_get_transform(self):
         q = CustomModel.objects.filter(field__transformfunc_banana=3)
-        self.assertIn('banana()', str(q.query))
+        self.assertRegex(str(q.query), r"'?banana'?\(\)", str(q.query))
 
     def test_overridden_get_lookup_chain(self):
         q = CustomModel.objects.filter(field__transformfunc_banana__lookupfunc_elephants=3)
-        self.assertIn('elephants()', str(q.query))
+        self.assertRegex(str(q.query), r"'?elephants'?\(\)", str(q.query))
 
     def test_overridden_get_transform_chain(self):
         q = CustomModel.objects.filter(field__transformfunc_banana__transformfunc_pear=3)
-        self.assertIn('pear()', str(q.query))
+        self.assertRegex(str(q.query), r"'?pear'?\(\)", str(q.query))
```

Comment:
We discussed these tests in the #10568 (see [comment](https://github.com/django/django/pull/10568#pullrequestreview-296832467)). I think we should improve them in advance.

Reply:
I think we should apply diff from my comment. Passing function names in `params` cannot work properly because drivers quote them, so `SQLFuncMixin` is incorrect. You should no longer be force to adjust these tests after
```diff
diff --git a/tests/custom_lookups/tests.py b/tests/custom_lookups/tests.py
index 5d4897c130..343465eaa6 100644
--- a/tests/custom_lookups/tests.py
+++ b/tests/custom_lookups/tests.py
@@ -133,7 +133,7 @@ class Exactly(models.lookups.Exact):
 
 class SQLFuncMixin:
     def as_sql(self, compiler, connection):
-        return '%s()', [self.name]
+        return '%s()' % self.name, []
 
     @property
     def output_field(self):
```
```